pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -54,7 +63,10 @@ import (
 // can be taking a fraction of the size of connection store to approximate the
 // number of expired connections, while having a min and a max to handle edge cases,
 // e.g. min(50 + 0.1 * connectionStore.size(), 200)
-const maxConnsToExport = 64
+const (
+	maxConnsToExport = 64
+	socketPath       = "/var/log/antrea/networkpolicy/suricata_eve.socket"
```

Comment:
I don't think `/var/log` makes sense for this

Reply:
I have changed it to "/var/run/suricata/suricata_eve.socket"